packages/plugin-compiler/src/main/kotlin/io/realm/compiler/AccessorModifierIrGeneration.kt

```diff
@@ -377,17 +377,17 @@ class AccessorModifierIrGeneration(private val pluginContext: IrPluginContext) {
                                 putValueArgument(0, irGet(receiver))
                                 putValueArgument(1, irString(property.name.identifier))
 
-                                // Check if the collection type is a subclass of RealmObject
-                                // If so, a flag needs to be added as a parameter
-                                // This is due to Kotlin Native being unable to retrieve the related
-                                // supertypes in runtime
-                                if (collectionType != CollectionType.NONE) {
-                                    val supertypes = (type as IrSimpleType).classifier.descriptor
-                                        .typeConstructor.supertypes
-                                    if (inheritsFromRealmObject(supertypes)) {
-                                        putValueArgument(2, irBoolean(true))
-                                    }
-                                }
+//                            // Check if the collection type is a subclass of RealmObject
```

Comment:
Why are this commented out?

Reply:
According to @edualonso (who did the initial prototype of this PR) we don't longer need this parameter as the `RealmObject` case is now handled as a fallback if the argument is not a primitive type.